charts/aws-efs-csi-driver/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-efs-csi-driver
-version: 2.0.0
-appVersion: 1.2.1
+version: 2.1.0
```

Comment:
we have helm changelog as well now, release process will need a slight update. https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/charts/aws-efs-csi-driver/CHANGELOG.md